CONTRIBUTING.md

```diff
@@ -11,3 +11,19 @@ Read and follow the coding style and guidelines [from this file](CODING_STYLE_GU
 - Don't forget to update any impacted CHANGELOG file(s)
 - Run the *fix_files.sh* script
 - Start a github pull request
+
+## To run `fix_files.sh` on MAC
+
+* Update `bash` from 3.x to latest (5.x)
+
+```bash
+brew install bash
+echo "/usr/local/bin/bash" | sudo tee -a /etc/shells
+chsh -s /usr/local/bin/bash
```

Comment:
I'll add a section based on your ideas, thanks for helping macOS to easily install a **usable** version of bash :)
Although I won't have them use chsh without informing that it will alter the active login shell (I'll add a comment for what this command does, especially now that the default macOS is no longer bash)

Reply:
could you propose this section here? I'll edit it in my PR, so you can merge

Reply:
I'll try as soon as I get some time.